TelCoColorCoder.cpp

```diff
@@ -0,0 +1,48 @@
+#include "TelCoColorCoder.h"
+
+namespace TelCoColorCoder
+{
+const char* MajorColorNames[] = { "White", "Red", "Black", "Yellow", "Violet"  };
+const char* MinorColorNames[] = { "Blue", "Orange", "Green", "Brown", "Slate"  }; ;
+    
+int numberOfMajorColors =  sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
+int numberOfMinorColors =  sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
+
+ColorPair GetColorFromPairNumber(int pairNumber)
+{
+        int zeroBasedPairNumber = pairNumber - 1;
+        MajorColor majorColor = (MajorColor)(zeroBasedPairNumber / numberOfMinorColors);
+        MinorColor minorColor = (MinorColor)(zeroBasedPairNumber % numberOfMinorColors);
+        return ColorPair(majorColor, minorColor);
+}
+
+int GetPairNumberFromColor(MajorColor major, MinorColor minor) 
+{
+        return major * numberOfMinorColors + minor + 1;
+};
+
+void Display_PairNumber_Major_minorColor()
+{ 
+     int pairNumber=1;
+     for(;pairNumber<26;pairNumber++)
```

Comment:
Avoid hard-coding numbers like 26. It's easy to forget updating it when the number of colors change

Reply:
Hi sir,
        Thank you so much for the feedback, I have update the code with constant.